pommerman/network/ionc/__init__.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+"""IonClient Entry point"""
+
+# Global Imports
+import ui
+from .constants import LangEN as Strings
+from .network import Network as net
+from signal import signal, SIGINT
+from gym.spaces import Discrete
+
```

Comment:
This is confusing:
1. Strings is a package in python. 
2. Strings is a complete renaming of LangEN. 
3. We no longer know where Strings comes from.

Suggestions:
1. from . import constants (then later do constants.LangEN).
2. from . import network (then later do network.Network)
3. from . import signal (then later do signal.signal)

Reply:
I think importing everything from a package is redundant so importing just what you need isn't a bad practice.
```python
import .constants # Imports everything from constants
Strings=constants.LangEN # We use just a subset of that
```
or
```python
from .constants import LangEN as Strings # Cleaner and imports only what we require
```
In addition to that I'd say it's rather simple to understand the origin of Strings just by reading the statment as plain english, "It has to import LangEN from constants under the name Strings".